docs/developer_guide/contributing.rst

```diff
@@ -0,0 +1,420 @@
+Contributing to Plankton
```

Comment:
I think I agree with renaming the document, but I also think that everything in that document belongs together...and especially with the anchored subsections it's still okay to navigate.